docker/vrx_trial/vrx_trial_install

```diff
@@ -19,9 +19,10 @@ MIL_REPO_DIR="$CATKIN_SOURCE_DIR/mil"
 mil_clone_repo()
 {
   sudo apt install -y git
-  git clone -b ${BRANCH} --single-branch ${GIT_URL} ${MIL_REPO_DIR}
+  git clone ${GIT_URL} ${MIL_REPO_DIR}
   cd ${MIL_REPO_DIR}
   git submodule update --init --recursive
```

Comment:
init submodule should happen after git reset so it fetches the correct submodules